app/containers/ItunesProvider/ItunesGridContainer/index.js

```diff
@@ -50,6 +52,23 @@ const ItunesGridContainer = ({ dispatchSongName, dispatchClearSongs, songs, erro
   };
   const debouncedHandleOnChange = debounce(handleOnChange, 200);
 
+  const renderSongs = () => {
+    if (!songs) {
+      return <div></div>;
+    }
+    if (songs.results) {
+      const songsGrid = songs.results.map((song, songIdx) => {
```

Comment:
its better if we don't have a const and then reuse it somewhere. We can just use the `For` component